controls/models.py

```diff
@@ -61,6 +62,41 @@ def catalog_control_as_dict(self):
         # Look up control by ID
         return catalog_control_dict[self.sid]
 
+    def create_prototype(self):
+        """Creates a prototype statement from an existing statement and prototype object"""
+
+        if self.prototype is not None:
+            # Prototype already exists for statement
+            return self.prototype
+            # check if prototype content is the same, report error if not, or overwrite if permission approved
+        from copy import deepcopy
```

Comment:
Unless you are doing a conditional import, you may want to consider placing all imports at the top of the file